ops/services/20-microservices/main.tf

```diff
@@ -155,30 +180,34 @@ resource "aws_security_group" "internal_lb" {
 
 resource "aws_security_group_rule" "lambda_sg_ingress_access" {
   type                     = "ingress"
-  from_port                = 80
-  to_port                  = 80
+  from_port                = 443
+  to_port                  = 443
   protocol                 = "tcp"
-  description              = "inbound access for lambda to microservices"
+  description              = "inbound enc access for lambda to microservices"
   source_security_group_id = data.aws_security_group.lambda.id
   security_group_id        = aws_security_group.internal_lb.id
 }
 
 resource "aws_security_group_rule" "api_sg_ingress_access" {
   type                     = "ingress"
-  from_port                = 80
-  to_port                  = 80
+  from_port                = 443
+  to_port                  = 443
   protocol                 = "tcp"
-  description              = "inbound access for microservices"
+  description              = "inbound enc access for microservices"
   source_security_group_id = data.aws_security_group.api.id
   security_group_id        = aws_security_group.internal_lb.id
 }
 
 resource "aws_security_group_rule" "worker_sg_ingress_access" {
   type                     = "ingress"
-  from_port                = 80
-  to_port                  = 80
+  from_port                = 443
+  to_port                  = 443
   protocol                 = "tcp"
-  description              = "inbound access for microservices"
+  description              = "inbound enc access for microservices"
   source_security_group_id = data.aws_security_group.worker.id
   security_group_id        = aws_security_group.internal_lb.id
 }
+
+data "aws_acm_certificate" "this" {
+  domain = local.parent_env == "prod" ? "api.ab2d.cms.gov" : "${local.parent_env}.ab2d.cms.gov"
+}
```

Comment:
Hmm, so we're using the same certificate for the internal and external load balancers? I suppose that's fine as long as nothing is checking the validity of the certificate on the internal load balancer. I think we need to discuss further about which certificates we're using across these internal resources.